lib/resolvers/project.js

```diff
@@ -202,6 +210,8 @@ module.exports = ({ models }) => async ({ action, data, id, meta = {} }, transac
       patch.expiryDate = start.add({ months, years }).toISOString();
     }
 
+    patch.raDate = getRaDate(versionToGrant, patch.expiryDate || versionToGrant.expiryDate);
```

Comment:
The expiry date will be on the project, not the version, right?
```suggestion
    patch.raDate = getRaDate(versionToGrant, patch.expiryDate || project.expiryDate);
```

Reply:
ooh good spot 👍